plugins/modules/aci_fabric_node.py

```diff
@@ -243,10 +258,19 @@ def main():
     switch = module.params.get("switch")
     description = module.params.get("description")
     role = module.params.get("role")
+    node_type = module.params.get("node_type")
+    remote_leaf_pool_id = module.params.get("remote_leaf_pool_id")
     state = module.params.get("state")
     name_alias = module.params.get("name_alias")
 
     aci = ACIModule(module)
+
+    if node_type is not None:
+        node_type = NODE_TYPE_MAPPING.get(node_type)
+
+    if remote_leaf_pool_id is not None:
+        node_type = NODE_TYPE_MAPPING.get("remote")
```

Comment:
If node_type `remote` is added I think this logic could be simplified, and the mutually_exclusive could be catch in a module fail.